schema/README.md

```diff
@@ -0,0 +1,32 @@
+# JSON schema
+
+## Overview
+
+This directory contains the [JSON Schema](http://json-schema.org/) for
+validating the `config.json` of this container runtime specification.
+
+The layout of the files is as follows:
+* [schema.json](./schema.json) - the primary entrypoint for the whole schema document
```

Comment:
Pedantic nit: the other link targets don't have a leading `./`, so maybe remove this one for consistency?